indexer/db/db.go

```diff
@@ -282,9 +249,9 @@ func (d *Database) AddIndexedL2Block(block *IndexedL2Block) error {
 
 	const insertWithdrawalStatement = `
 	INSERT INTO withdrawals
-		(guid, from_address, to_address, l1_token, l2_token, amount, tx_hash, log_index, l2_block_hash, data)
+		(guid, from_address, to_address, l1_token, l2_token, amount, tx_hash, log_index, block_hash, data, br_withdrawal_hash)
```

Comment:
Why does this need to be tying bedrock into the withdrawal hash? It seems like its not very future proof

Reply:
It delineates which fields are legacy, and which are not. This field is only used for Bedrock withdrawals. We can remove the prefixes after launch once we’re no longer supporting the legacy system.